pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -536,4 +539,28 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = InfraSAs.addServiceAccount(
+		InfraGCControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: GCControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// GCController.podStore.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
```

Comment:
I don't feel strongly... some of the controllers took a lot of digging, and I anticipated tightening permissions eventually. I would probably only add comments to permissions we don't think are reasonable so we can track when it might be ok to remove them